pydarn/plotting/fan.py

```diff
@@ -268,6 +270,9 @@ def plot_fov(cls, stid: str, dtime: dt.datetime, ax=None,
             rs - radius polar coordinates
         """
         # Get radar beam/gate locations
+        if ranges == []:
+            max_ranges = SuperDARNRadars.radars[stid].hardware_info.gates
+            ranges = [0, max_ranges]
         beam_corners_aacgm_lats, beam_corners_aacgm_lons = \
             radar_fov(stid, coords='aacgm', date=dtime)
         fan_shape = beam_corners_aacgm_lons.shape
```

Comment:
```suggestion
        fan_shape = beam_corners_aacgm_lons.shape
        
        if ranges == []
            ranges = [0, fan_shape[1]]
```

Reply:
I will use this bandage to replace the actual necessary solution where the radar FOV files need to be consistent with the hardware files.